ui-src/src/app/configurations/configurations.component.html

```diff
@@ -1,8 +1,16 @@
 <div class="container">
+    <div class="row">
+        <div class=" column col-xs-8 col-sm-8 col-md-8 col-lg-8 ">
+            <label>Config File Path *</label>
+            <input class="form-control" type="string" [(ngModel)]="configPath" id="config-path" required>
+        </div>
+        <div class=" column col-xs-4 col-sm-4 col-md-4 col-lg-4 ">
+            <label>Xml File Name *</label>
+            <input class="form-control" type="string" [(ngModel)]="xmlFileName" id="xml-file-name" required>
+        </div>
+
+        <button class="fileButton btn1 btn-primary1 pull-right" id="btn-get" (click)="onGetFile()">Get File</button>
```

Comment:
I thought this was meant to be automatic and there would be no need for human intervention?

Reply:
No we decided that we use a get file button as it would be annoying if a error message appeared if you accidentally clicked off the box without completing the file name

Reply:
ok, that makes sense.